packages/cli/src/command-build.js

```diff
@@ -167,11 +167,20 @@ async function executeDuringPhase(
   biltin,
   dryRun,
 ) {
-  return await buildPackages(
-    finalPackagesToBuild,
-    makePackageBuild(jobConfiguration, rootDirectory, buildOptions, biltin),
-    dryRun,
-  )
+  /**@return {import('@bilt/build').BuildPackageFunction} */
+  const buildCounter = () => {
```

Comment:
If `pointer` does not need to be passed (see my other comment below), then there's no real need for _this_ function.

Reply:
the pointer is passing down - I use closer to increment it